SU2_CFD/include/drivers/CDriver.hpp

```diff
@@ -525,7 +525,7 @@ class CDriver {
    * \param[in] iVertex - Vertex identifier.
    * \return Unit normal (vector) at the vertex.
    */
-  vector<passivedouble> GetVertexUnitNormal(unsigned short iMarker, unsigned long iVertex) const;
+  vector<passivedouble> GetVertexNormal(unsigned short iMarker, unsigned long iVertex, bool unitNormal) const;
```

Comment:
```suggestion
  vector<passivedouble> GetVertexNormal(unsigned short iMarker, unsigned long iVertex, bool unitNormal = false) const;

  inline vector<passivedouble> GetVertexUnitNormal(unsigned short iMarker, unsigned long iVertex) const {
    return GetVertexNormal(iMarker, iVertex, true);
  }
```

IDK if swig understands inline methods (?) but it would be good to maintain some stability of the python interface.
(unless this method (GetVertexUnitNormal) was new and never released before).

Reply:
I see your point and agree with you